packages/example-data/src/facepile.ts

```diff
@@ -1,3 +1,5 @@
+import * as React from 'react';
```

Comment:
This package doesn't currently depend on `@types/react`--the code only happens to compile because React is present in the monorepo (and yarn isn't strict about implicit dependencies). So if we decide to go forward with this, you should add `@types/react` as a dev dependency and peer dependency of example-data, similar to other packages.